css-backgrounds-4/Overview.bs

```diff
@@ -895,15 +903,13 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either the keyword ''box-shadow-offset/none'' or
-    a list, each item consisting of four absolute lengths
-    plus a computed color and optionally also a ''box-shadow-position/inset'' keyword
+Computed value: see individual properties
 Animatable: by computed value,
-    treating ''box-shadow-offset/none'' as a zero-item list
-    and appending blank shadows (''transparent 0 0 0 0'')
-    with a corresponding ''box-shadow-position/inset'' keyword as needed
-    to match the longer list
-    if the shorter list is otherwise compatible with the longer one
+	letting ''box-shadow-offset/none'' create a blank shadow
+	(''transparent 0 0 0 0'') when interpolated with non-'none' values
+	and appending blank shadows with a corresponding
+	''box-shadow-position/inset'' keyword as needed to match the longer list
+	if the shorter list is otherwise compatible with the longer one
```

Comment:
I don't think this works.

Let's say we interpolate from `box-shadow-offset: none; box-shadow-color: #f00` to `box-shadow-offset: 10px 10px; box-shadow-color: #f00`.

`box-shadow-offset` interpolates `none` as `0 0`, so at 50% we have `5px 5px`. But what about `box-shadow-color`? Its computed value was always `#f00` (`none` was only changing the used value into `transparent`), so we get `#f00`, but current browsers say `rgba(255, 0, 0, 0.5)`.

So I guess that either:

 - `box-shadow-offset: none` should affect the computed values of other properties (even though coordinated value lists were only supposed to affect each other at used value time I believe).
 - And interpolation with `box-shadow-offset: none` should not just treat it as `0 0` but somehow affect the used value of other properties at used value time.
 - We get rid of this magical behavior, as you initially wanted.